chapters/features.adoc

```diff
@@ -5887,6 +5887,34 @@ include::{generated}/validity/structs/VkPhysicalDeviceHostImageCopyFeaturesEXT.a
 --
 endif::VK_EXT_host_image_copy[]
 
+ifdef::VK_EXT_present_timing[]
+[open,refpage='VkPhysicalDevicePresentTimingFeaturesEXT',desc='Structure indicating support for present timing',type='structs']
+--
+The sname:VkPhysicalDevicePresentTimingFeaturesEXT structure is defined as:
+
+include::{generated}/api/structs/VkPhysicalDevicePresentTimingFeaturesEXT.adoc[]
+
+This structure describes the following feature:
+
+  * pname:sType is the type of this structure.
+  * pname:pNext is `NULL` or a pointer to a structure extending this
+    structure.
+  * [[features-presentTiming]] pname:presentTiming indicates that the
+    implementation supports fname:vkGetPastPresentationTimingEXT.
+  * [[features-presentAtAbsoluteTime]] pname:presentAtAbsoluteTime indicates
+    that the implementation supports specifying present times with
+    sname:VkPresentTimeEXT::pname:targetPresentTime.
+  * [[features-presentAtRelativeTime]] pname:presentAtRelativeTime indicates
+    that the implementation supports specifying relative present times with
+    sname:VkPresentTimeEXT::pname:minPresentDuration.
```

Comment:
same here: Should this "minPresentDuration" actually be "presentDuration" or targetPresentTime?